packages/client/components/MeetingSubnavItem.tsx

```diff
@@ -31,8 +31,8 @@ const ItemRoot = styled('div')<ItemRootProps>(
     position: 'relative',
     userSelect: 'none',
     width: '100%',
-    '&:hover': {
-      backgroundColor: onClick && !isActive ? PALETTE.SLATE_100 : undefined,
+    '&:hover, :focus': {
```

Comment:
-1 Is there a reason you've added `focus` here? It appears to me as if this could conflict with `isActive`.